apps/condo/next.config.js

```diff
@@ -55,7 +55,7 @@ const apolloBatchingEnabled = !falsey(conf['APOLLO_BATCHING_ENABLED'])
 const tourVideoUrl = JSON.parse(conf['TOUR_VIDEO_URL'] || '{}')
 const residentAppLandingUrl = JSON.parse(conf['RESIDENT_APP_LANDING_URL'] || '{}')
 const createMapVideoUrl = JSON.parse(conf['CREATE_MAP_VIDEO_URL'] || '{}')
-const guideModalCardLink = JSON.parse(conf['GUIDE_MODAL_CARD_LINK'] || '{}')
+const guideModalCardExamples = JSON.parse(conf['GUIDE_MODAL_CARD_EXAMPLES'] || '{}')
```

Comment:
where variable is `GUIDE_MODAL_CARD_EXAMPLES`?

Reply:
Okey, how I should name this variable?

Reply:
Where is this variable in `.helm`? I didn't find it

Reply:
In `.helm` I am waiting for approval of the changes

Reply:
How I can this name this banner? 
<img width="1263" alt="image" src="https://github.com/user-attachments/assets/483ec011-0008-43b9-94d0-331464edc874">

Reply:
just idk managementCompanyReview banner

Reply:
`guideManagementCompanyReview` this name be better? @Alllex202

Reply:
> In `.helm` I am waiting for approval of the changes

Add these changes to this pr.
You should switch the submodule to your commit with changes

Reply:
> `guideManagementCompanyReview` this name be better? @Alllex202

I don't think it's related to company reviews. Look where all the links listed lead.
These are just links to blog posts. Some with company reviews, some without

Reply:
> `guideManagementCompanyReview` this name be better? @Alllex202

How about `guideModalCardReviews`?